app.py

```diff
@@ -23,11 +26,21 @@ def create_app():
     app = Flask(__name__) 
 
     login_disabled = os.environ.get('LOGIN_DISABLED', 'False') == 'True'
+    log_level = os.environ.get('LOG_LEVEL', 'INFO')
+    loggly_token = os.environ.get('LOGGLY_TOKEN')
```

Comment:
We should add some reference to these in our documentation, i.e. in the Readme a reference to Loggly, and add the keys to our `.env.template` file

Reply:
done